docs/source/getting_started/index.rst

```diff
@@ -1,10 +1,27 @@
 Getting started
-#######################################
+##############################################################################
 
-The following pages take you step by step with getting started with the CS Unplugged project. Start here if you’re new to the project.
+This documentation will help you understand how the project is setup,
+basic steps on how to use it, and our guidelines for your contributions.
+
+.. note::
+
+  This project adheres to the Contributor Covenant code of conduct.
+  By participating, you are expected to uphold this code.
+  Please read our :doc:`Code of Conduct <code_of_conduct>` before continuing.
+  You can report unacceptable behavior to
```

Comment:
We prefer the non-American spelling which is `behaviour`, so I'll fix that.

But sadly I copied it from the Kordac docs so you will need to change it there too. 😄